translations/nl.ts

```diff
@@ -239,55 +239,91 @@ Proxy omzeilen en een directe internetverbinding gebruiken?</translation>
         <translation>Formulier</translation>
     </message>
     <message>
-        <location filename="../src/android.ui" line="531"/>
+        <location filename="../src/android.ui" line="252"/>
+        <source>qTox</source>
+        <translation>qTox</translation>
+    </message>
+    <message>
+        <location filename="../src/android.ui" line="343"/>
+        <source>Someone</source>
+        <translation>Iemand</translation>
+    </message>
+    <message>
+        <location filename="../src/android.ui" line="348"/>
+        <source>Someone else</source>
+        <translation>Iemand anders</translation>
+    </message>
+    <message>
+        <location filename="../src/android.ui" line="353"/>
+        <source>Groupbot</source>
+        <translation>Groupbot</translation>
+    </message>
+    <message>
+        <location filename="../src/android.ui" line="358"/>
+        <source>That guy who I don&apos;t remember adding</source>
+        <translation>Degene die ik me niet kan herinneren toegevoegd te hebben</translation>
```

Comment:
Sorry, it's done automatically by the Designer tool. I'll be more careful next time.